lib/gh/gh.ml

```diff
@@ -1,2 +1,4 @@
+module Json = Yojson.Basic.Util
```

Comment:
I think this line can be removed from this file since it's not exposed by the corresponding `.mli` file